CHANGELOG.md

```diff
@@ -36,9 +36,13 @@ Ref: https://keepachangelog.com/en/1.0.0/
 # Changelog
 
 ## [Unreleased]
+* (modules/nft)  [\#243](https://github.com/irisnet/irismod/pull/243) Refactor nft module with cosmos-sdk x/nft
 
 ### Improvements
 
+### API Breaking Changes
+* (modules/nft)  [\#243](https://github.com/irisnet/irismod/pull/243) GRPC interface `Owner` query conditions have been changed, the original parameter `Owner `is mandatory,`DenomID` is optional, now `DenomID` is mandatory, `Owner` is optional
```

Comment:
The query `Owner` is removed, and replaced with `NFTsOfClass`, of which the parameter `DenomID` is mandatory and `Owner` is optional